src/backends/backend.js

```diff
@@ -26,7 +26,24 @@ class LocalStorageAuthStore {
 
 const slugFormatter = (template = "{{slug}}", entryData) => {
   const date = new Date();
-  const identifier = entryData.get("title", entryData.get("path"));
+
+  const getIdentifier = (entryData) => {
+	  const validIdentifierFields = ["title", "path"];
+	  const identifiers = validIdentifierFields.map((field) => {
+	  	return entryData.find((_, key) => {
+	  		return key.toLowerCase() === field;
+	  	});
+	  });
+	  
+	  const identifier = identifiers.find(i => typeof i !== 'undefined');
+
+	  if (typeof identifier === 'undefined') {
+	      throw new Error("Collection must have a field name that is a valid entry identifier"); 
+	  } else {
```

Comment:
There's no need for this `else`, as execution will not proceed pass the `if` block if the condition is true. Instead, you can just `return identifier;` at the end of the function.